lib/model/game.dart

```diff
@@ -1,13 +1,16 @@
 import 'package:flutter/material.dart';
 
 typedef Board = List<List<CellState>>;
+typedef Coordinates = ({int x, int y})?;
```

Comment:
Let's make the type not-nullable. 
And notifier will be ValueNotifier<Coordinates?>
This way it will be more readable

Reply:
done